### ISSUE TITLE
Warping location

### ISSUE THREAD

**Issue by moepi2k** [suriyun-production/mmorpg-kit-docs#2679]
player reporting that they randomly warping to wrong location. happen to me also once. instead of target location in warpportalentity it portet to maps 0, 0,0 location. its as a said rarely and random but happen. all my warp stuff is def untouched since 188 (before update it never happened) so i cant see any changes only one thing i gues the scene object thing. because my warp portal are all scene objects.

**Comment by insthync** [comment]
I don't think it is, if you guess that then you may change change it as you wish

**Comment by moepi2k** [comment]
Hm anything else related changed? Player warp iut of map.sometimes.. but
some thing im.really sure is the new litenetlib thing is, that all my
disabled scene object automaticly gets activated.

Ittipon Teerapruettikulchai ***@***.***> schrieb am Mo., 23.
Dez. 2024, 03:21:

**Comment by insthync** [comment]
Maybe character saving, how your teleport position being changed, can you answer me?

**Comment by moepi2k** [comment]
what do you mean? or what should i check?

**Comment by insthync** [comment]
How is your warp portal setup?, is it teleport to another map?

**Comment by moepi2k** [comment]
yes simple warpportalenttiy. add mapinfo to another nap and position. but a player just reported that it happened now when he died. instead of teleported back to spawn position it also warped to destination map but instead of correct position it ported to 0,0,0

**Comment by insthync** [comment]
What? it is teleporting by warp portal or respawning?

**Comment by moepi2k** [comment]
both. player reported it now for both. when dying and teleport back and when using warpportalentity

**Comment by insthync** [comment]
If it both then it is not because of scene object, isn't it, I think it is because of character saving, because I made changes to it to make it save more frequently but smaller data, (not 100% done yet)

**Comment by kaal86** [comment]
Ok good to know thx. But why it teleport to wrong location when use warp portal. The locTiin is static set on the warp portal

**Comment by moepi2k** [comment]
any solution for the warping/respawning issue? its annyoing. so many player report that they spawn/warp in wrong location...

**Comment by insthync** [comment]
I don't know what is the cause of problem yet, I can't reproduce yet. Is it really occuring did you ever saw it by your eyes?

**Comment by insthync** [comment]
I've made some update I can create a new release now, but it is not stable yet, and may have other problems, will you try it?

**Comment by moepi2k** [comment]
yes after updating to latest version it occured 2 times to me in 30 minutes. its def something from the new update. but i cant find what. no its okey. take ur time for the update. better to release full working. just wanted to let u know so u dont forget it. 

i thought maybe u know whats the issue because of recent changes so i could manualy change code to test. but i can wait
